dotCMS/src/main/java/com/dotcms/rest/api/v2/languages/LanguagesResource.java

```diff
@@ -135,6 +138,71 @@ public final Response saveLanguage(@Context final HttpServletRequest request,
         return Response.ok(new ResponseEntityView(language)).build(); // 200
     }
 
+    @POST
+    @JSONP
+    @NoCache
+    @Path("/{languageTag}")
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response saveFromLanguageTag(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response,
+            @PathParam("languageTag") final String languageTag
+    ) {
+        DotPreconditions.notNull(languageTag, "Expected languageTag Param path was empty.");
+        this.webResource.init(null, request, response,
+                true, PortletID.LANGUAGES.toString());
+
+        final Locale locale = validateLanguageTag(languageTag);
+
+        final LanguageForm languageForm = new LanguageForm.Builder()
+                .language(locale.getDisplayLanguage()).languageCode(locale.getLanguage())
+                .country(locale.getDisplayCountry()).countryCode(locale.getCountry()).build();
+
+        final Language language = saveOrUpdateLanguage(null, languageForm);
```

Comment:
the saveOrUpdateLanguage, should be able to validate if do not exists any language with that combination
see here for more reference: https://github.com/dotcms/core/issues/7342

Reply:
done